src/main/java/org/broadinstitute/hellbender/tools/spark/CreateHadoopBamSplittingIndex.java

```diff
@@ -32,6 +34,7 @@
 @CommandLineProgramProperties(summary = "Create a HadoopBam splitting index as well as a bam index from a bam file.",
         oneLineSummary = "create a hadoop-bam splitting index" ,
         programGroup = SparkProgramGroup.class)
+@DocumentedFeature(groupName= HelpConstants.DOCS_CAT_DATA)
```

Comment:
I made up all these changes (the annotations, groups, etc.) and kept them in a separate commit since I needed a range of interesting cases (i.e., Spark tools, Walkers, Picard tools, ReadFilters) to use to exercise the code, but thinking we might not actually take them in this PR. We could leave them out, but without them, the doc process doesn't output anything, and they at least provide a jump start.

The question of how we decide what goes in Spark is an interesting one. We really have only one dimension of partitioning (if you don't count supercat - I'll save that for the other comment), but we might want more than one dimension so we can have Spark/NonSpark, and within those, the normal categories. As @vdauwera said, I think its worth doing thinking a bit about how we want to organize the doc before we go too much further.